AdofaiTweaks/Tweaks/PlanetColor/PlanetColorPatches.cs

```diff
@@ -14,18 +14,20 @@ internal static class PlanetColorPatches
         [SyncTweakSettings]
         private static PlanetColorSettings Settings { get; set; }
 
-        private static bool IsRedPlanet(scrPlanet planet) {
-            return planet == scrController.instance.redPlanet;
+
+        // When you complain about how hideous this is just know that I hate it as much as you do
+        private static bool IsRedPlanet(PlanetRenderer planet) {
+            return planet == scrController.instance.planetRed.planetRenderer;
```

Comment:
could have received `scrPlanet` instance, and then compared `.planetRenderer`. I think that just makes more sense.

Reply:
ok actually that would be much less performant because then i would have to get the parent that doesn't exist in the field